src/test/java/automation/utils/loaders/EnvironmentConfigLoader.java

```diff
@@ -23,11 +23,25 @@ public WebDriver getDriver() {
         return new FirefoxDriver();
     }
 
-    public String getURL() {
+    public String urlBeginning() {
         // fixme check nulls
+        // fixme write buider for making URL?! do I still need this if I move urls to Pages enum?
         String hostname = properties.getProperty("hostname");
         String port = properties.getProperty("port");
         String pageURL = properties.getProperty("pageURL");
         return String.format("http://%s:%s%s", hostname, port, pageURL );
     }
+
+    public String getGroupManagementPage() {
+        String adminBrowsePanel = properties.getProperty("adminBrowsePanel");
+        String adminToolsPanel = properties.getProperty("adminToolsPanel");
+        return String.format("%s%s%s", urlBeginning(), adminToolsPanel, adminBrowsePanel);
```

Comment:
You wouldn't need to complicate methods in this class if you use simple string concatenation in environment.properties: https://stackoverflow.com/questions/25003341/properties-file-variable-concatenation

Reply:
Solved this with ApacheCommons